docs/concepts/glossary.md

```diff
@@ -161,6 +161,6 @@ subsequent Tasks.
 A Transformation is applied on one or more data streams or data sets and results in one or more
 output data streams or data sets. A transformation might change a data stream or data set on a
 per-record basis, but might also only change its partitioning or perform an aggregation. While
-[Operators](#operator) and [Functions](#function)) are the "physical" parts of Flink's API,
-Transformations are only an API concept. Specifically, most - but not all - transformations are
```

Comment:
Was that wrong, or is this mainly a language improvement?

Reply:
Language improvement. "most" already implies "not all", hence this is redundant and does not bring any additional information. Additionally I believe it should be separated with the commas, not hyphens from the punctuation perspective.

Reply:
LGTM